src/Request.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs')
+
+class Request {
+  constructor (event) {
+    this.request = this.createRequest(event)
+  }
+
+  getRequest () {
```

Comment:
`get request () {…}`

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

Reply:
Added getter and setter, and test for coverage